services/director-v2/src/simcore_service_director_v2/modules/director_v0.py

```diff
@@ -23,7 +32,8 @@ def setup(app: FastAPI, settings: DirectorV0Settings):
 
     def on_startup() -> None:
         DirectorV0Client.create(
-            app, client=httpx.AsyncClient(base_url=settings.base_url(include_tag=True))
+            app,
+            client=httpx.AsyncClient(base_url=settings.base_url(include_tag=True)),
```

Comment:
what about timeouts for requests forwarded via this client?

Reply:
it is using whatever defaults. I'll have a look at the httpx thing. your timeout are widely available right?

Reply:
If you are referring to the timeout pull request. No it was not yet merged, still not approved. I needed some feedback for it.

Reply:
if you don't mind I would keep them as is for now. since the ones for the upload still go directly from the webserver to the director I guess it will be unaffected for now. Since this PR might go in before yours, the timeout standards might still evolve. then we can add them in there.